### PR TITLE
Python27fixes

### PR DESCRIPTION
**Pull request by danibudi**
@boyankukushev 
please review the fixes related to py 2.7
also the SALT can be configured as environment variable or in local_settings.py /Django, Flask friendly/
also it's implemented check for the correct SALT /if not - error reported/

### CODE REVIEW

File: kount/request.py [hunk 1]
```diff
@@ -130,6 +130,11 @@ def __init__(self):
         self.params["SDK"] = "python"
         self.payment = None
         self.close_on_finish = None
+        card_solted = Khash.hash_payment_token(token="666666669")
```
Comment (boyankukushev) [inline kount/request.py]: So, this is not correct in terms of Kount-provided data.

Please, check if the salt is correct against the sha256 of the salt (value = https://github.com/Kount/kount-ris-java-sdk/blob/master/kount-ris-sdk/src/main/resources/correct.salt.crypt) and in the way it's happening here https://github.com/Kount/kount-ris-java-sdk/blob/master/kount-ris-sdk/src/main/java/com/kount/ris/util/Khash.java#L58

Thanks!

**Comment by boyankukushev** [comment]
Why is the salt value referred at so many places? Isn't it possible to just refer it at the place, responsible for actual khashing?

File: kount/request.py (hunk 1 above)
Comment (danibudi) [inline kount/request.py]: fixed in next commit. but in python good practices it's recommended to check with real example

**Comment by danibudi** [comment]
fixed in next commit

**Comment by danibudi** [comment]
please review the latests commit with fixed issues

**Pull request closed without merging**